h2o-r/h2o-package/R/automl.R

```diff
@@ -199,7 +199,19 @@ h2o.automl <- function(x, y, training_frame,
   #project <- automl_job$project  # This is not functional right now, we can get project_name from user input instead
   leaderboard <- as.data.frame(automl_job["leaderboard_table"]$leaderboard_table)
   row.names(leaderboard) <- seq(nrow(leaderboard))
-  leaderboard <- as.h2o(leaderboard)  # Convert to H2OFrame
+  
+  # Intentionally mask the progress bar here since showing multiple progress bars is confusing to users.
+  # If any failure happens, revert back to h2o.show_progress() and display the error message.
+  h2o.no_progress()
+  leaderboard <- tryCatch(
+    as.h2o(leaderboard),
+    error = function(e) {
+      h2o.show_progress()
+      stop(e$message)
+    }
+  )
+  h2o.show_progress()
```

Comment:
This overrides user setting. If the user disabled progress bar himself this will re-enable it

Reply:
Good point. I will update the PR to address that.